well-architected/mission-critical/index.yml

```diff
@@ -66,3 +66,11 @@ landingContent:
             url: https://github.com/azure/mission-critical-online
           - text: Mission-Critical Connected
             url: https://github.com/azure/mission-critical-connected
+
+ # Card
+  - title: Video library
+    linkLists:
+      - linkListType: video
+        links:
+          - text: Resiliency with Azure Load Test and Azure Chaos Studio
```

Comment:
Can we change the text ("Resiliency with Azure Load Test and Azure Chaos Studio") to read: "Continuous Validation with Azure Load Test and Azure Chaos Studio"

The url will also need changed as well

Reply:
Sure can!